roles/contiv_network/tasks/fwd_tasks.yml

```diff
@@ -0,0 +1,3 @@
+- name: set fwd mode
```

Comment:
sure. I was thinking the convention was to add it in seperate files similar to aci. and eventually if we end up passing some args and stuff to the global setting it might be cleaner. your call.